### PR TITLE
remove login button from formMain

### PR DESCRIPTION
**Pull request by arbelsh12**

### CODE REVIEW

File: FacebookWinFormsApp/FormLogIn.cs [hunk 1]
```diff
@@ -65,6 +65,11 @@ private void FormLogIn_Load(object sender, EventArgs e)
         {
 
         }
+
+        private void labelWelcome_Click(object sender, EventArgs e)
```
Comment (shanikrauss) [inline FacebookWinFormsApp/FormLogIn.cs:69]: Autogenerated?

File: FacebookWinFormsApp/FormMain.cs [hunk 2]
```diff
@@ -17,52 +17,23 @@ public partial class FormMain : Form
         private const int k_NotSelected = -1;
         private readonly Astrology r_Astrology;
         private readonly FilterEvents r_FilterEvents;
+        private readonly FormLogIn r_FormLogIn;
         private User m_LoggedInUser;
-        //private LoginResult m_LoginResult;
```
Comment (shanikrauss) [inline FacebookWinFormsApp/FormMain.cs:21]: why?

**Review by shanikrauss — COMMENTED** [review]
left some quastions

File: FacebookWinFormsApp/FormMain.cs (hunk 2 above)
Comment (arbelsh12) [reply FacebookWinFormsApp/FormMain.cs:21]: why what ?

File: FacebookWinFormsApp/FormLogIn.cs (hunk 1 above)
Comment (arbelsh12) [reply FacebookWinFormsApp/FormLogIn.cs:69]: yes

**Pull request merged**